### PR TITLE
Publish documentation to new Monax web site.

### PR DESCRIPTION
**Pull request by NodeGuy**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,42 +25,45 @@
...
-  "author": {
-    "name": "Andreas Olofsson",
-    "email": "andreas@erisindustries.com"
-  },
+  "license": "LGPL-3.0",
+  "main": "index.js",
   "maintainers": [
     {
       "name": "David Braun",
       "email": "David@ErisIndustries.com"
```
Comment (zramsay) [inline package.json]: `@monax.io`

File: package.json [hunk 2]
```diff
@@ -23,42 +25,45 @@
     "bundle-dependencies": "1.0.2",
```
Comment (zramsay) [inline package.json]: line 22 (GH isn't letting me comment directly on it) for importing web3 => that commit has been merged :) into web3.js and released as 0.18.0 as of 6 hours ago ... should it be updated?

File: package.json (hunk 1 above)
Comment (NodeGuy) [inline package.json]: Nice catch; fixed.

File: package.json (hunk 2 above)
Comment (NodeGuy) [inline package.json]: It will be in the future; that's in my queue.

Comment (zramsay) [inline package.json]: :+1:

**Pull request merged**